/maneuvering:
  ros__parameters:
    distance_to_move: 30.0
    angle_from_target: 20.0
    camera_res_x: 640
    camera_fov: 70
    recovery_behaviour: STOP
    time_to_pause_search: 3.0
    time_between_recovery_actions: 2.0
    time_to_stop_before_recovery: 2.0
    red_buoy_label: red_buoy
    green_buoy_label: green_buoy
    second_red_buoy_label: black_buoy
    second_green_buoy_label: yellow_buoy
    frame_stack_size: 4
    bbox_selection: INNERMOST
    max_consec_recoveries: 2
/speed:
  ros__parameters:
    distance_to_move: 30.0
    angle_from_target: 20.0
    camera_res_x: 640
    camera_fov: 70
    recovery_behaviour: STOP
    time_to_pause_search: 3.0
    time_between_recovery_actions: 2.0
    time_to_stop_before_recovery: 2.0
    red_buoy_label: red_buoy
    green_buoy_label: green_buoy
    blue_buoy_label: blue_buoy
    second_blue_buoy_label: blue_buoy
    second_red_buoy_label: black_buoy
    second_green_buoy_label: yellow_buoy
    frame_stack_size: 4
    bbox_selection: INNERMOST
    use_start_point: true
    time_to_find_bay: 5.0
    max_time_between_bay_detections: 3.0
    max_time_between_buoy_detections: 3.0
    buoy_offset_angle: 0.0
    estimated_buoy_dist: 10.0
    buoy_circling_radius: 3.0
    num_pnts_on_semicircle: 4
    min_dist_from_bay_b4_return: 30.0
/task_controller:
  ros__parameters:
    tasks: ms
